configuration/Configuration-Debug.xcconfig

```diff
@@ -11,4 +11,5 @@ COPY_PHASE_STRIP = NO
 
 GCC_OPTIMIZATION_LEVEL = 0
 GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1 $(inherited)
+SIGNPOST_ENABLED = 1
```

Comment:
Shouldn't hat move into the previous line to be available to the preprocessor ?
```
GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1 SIGMPOST_ENABLED=1 $(inherited)
```

Reply:
+1